Package.swift

```diff
@@ -7,6 +7,7 @@ let package = Package(
     name: "SwiftRs",
     platforms: [
         .macOS(.v10_10), // OS X Yosemite is the minimum supported version
+        .iOS(.v11),
```

Comment:
Any particular reason why iOS 11 is the minimum?

Reply:
Well initially I wanted to use iOS 9 (just because it's the default value of cargo-mobile) but it warns me that iOS 11 is the actual minimum (maybe because of my macOS version?).